PowerballApi/Repositories/IRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace PowerballApi.Api.Repositories
+{
+    using System.Collections.Generic;
+    public interface IRepository<T> where T : class
+    {
+        List<T> Get();
+
+        T GetById(string id);
```

Comment:
FYI, you could just use Get(string Id) here, letting the parameters do the talking for you.